devel/pulp/devel/test_runner.py

```diff
@@ -12,6 +12,27 @@
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pulp.server.webservices.settings'
 
 
+# From: http://stackoverflow.com/questions/377017/test-if-executable-exists-in-python/377028#377028
```

Comment:
Thanks for the feedback. I'll write up a new version so that it can be incorporated into pulp without license issues.